R/mutate.R

```diff
@@ -429,7 +429,7 @@ mutate_bullets <- function(cnd, ...) {
 #' @export
 `mutate_bullets.dplyr:::mutate_incompatible_size` <- function(cnd, ...) {
   error_context <- peek_error_context()
-  error_name <- error_context$error_name
+  error_name <- ctxt_error_label(error_context)
```

Comment:
```suggestion
  label <- ctxt_error_label(error_context)
```

Or `error_label`?

This would just be for consistency with what the function is named. There are a few other places in this file you'd also have to do this renaming if you did this